### PR TITLE
Relax click dependency.

### PR DESCRIPTION
**Pull request by asvetlov**
Allow to install cherry_picker with the modern click version

### CODE REVIEW

**Comment by Mariatta** [comment]
Thanks! Do you need a release cut?

**Comment by asvetlov** [comment]
It would be nice!

**Comment by Mariatta** [comment]
Can you increase/update the version info, and the changelog?
Similar to https://github.com/python/core-workflow/pull/282/files

After that, create a GitHub tag of the release branch, and it will be automatically released to PyPI.

There's a brief write-up here: https://github.com/python/core-workflow/tree/master/cherry_picker#publishing-to-pypi

**Comment by Mariatta** [comment]
v1.2.2 is now on PyPI

File: cherry_picker/cherry_picker/cherry_picker.py [hunk 1]
```diff
@@ -381,11 +381,11 @@ def check_repo(self):
               help=("Path to config file, .cherry_picker.toml "
                     "from project root by default"),
               default=None)
-@click.argument('commit_sha1', 'The commit sha1 to be cherry-picked', nargs=1,
```
Comment (webknjaz) [inline cherry_picker/cherry_picker/cherry_picker.py]: Why remove the explanation? How is this related to the click dependency update?

Comment (webknjaz) [reply cherry_picker/cherry_picker/cherry_picker.py]: oh, they changed api..

Comment (asvetlov) [reply cherry_picker/cherry_picker/cherry_picker.py]: Explicitly forbade meaningless argument actually. The help was never displayed IIRC

Comment (webknjaz) [reply cherry_picker/cherry_picker/cherry_picker.py]: Oh, nevermind. I've already figured out that it fails with this arg under newer Click: https://travis-ci.org/python/core-workflow/jobs/491106736#L270
